packages/synapse-react-client/src/components/dataaccess/SubmissionPage/ApproveSubmissionConfirmationModal.tsx

```diff
@@ -0,0 +1,79 @@
+import { DialogBase } from '@/components/DialogBase'
+import { displayToast } from '@/components/index'
+import { useUpdateDataAccessSubmissionState } from '@/synapse-queries/index'
+import { Alert, Button, Typography } from '@mui/material'
+import { SubmissionState } from '@sage-bionetworks/synapse-types'
+import { useCallback } from 'react'
+
+type ApproveSubmissionConfirmationModalProps = {
+  open: boolean
+  submissionId: string
+  onClose: () => void
+}
+
+/**
+ * Confirmation modal to approve a data access submission asa a reviewer/ACT member.
+ */
+export function ApproveSubmissionConfirmationModal(
```

Comment:
Existing UI & logic for the submission page for reviewers to approve a submission. Logic was just refactored to be encapuslated in this component instead of inside the SubmissionPage component.